### PR TITLE
Add Scoop installation in README

### PR DESCRIPTION
**Pull request by sitiom**
Continuation of https://github.com/Watfaq/PowerSession/pull/48

### CODE REVIEW

**Comment by sitiom** [comment]
Ping @ibigbug (you don't seem to be watching this repo)

File: README.md [hunk 1]
```diff
@@ -19,10 +19,18 @@ PowerShell version of [asciinema](https://github.com/asciinema/asciinema) based
...
+### Cargo
+
 ```console
 cargo install PowerSession
 ```
 
+### Scoop
+
+```console
+scoop install powersession-rs
```
Comment (ibigbug) [inline README.md:31]: Is it a convention that the package name all be lowercases ?

Comment (sitiom) [reply README.md:31]: That seems to be the case for official buckets.

**Comment by ibigbug** [comment]
thanks!

**Pull request merged**